step-framework-parent/step-framework-collections-mongodb/src/main/java/step/core/collections/mongodb/MongoDBCollection.java

```diff
@@ -109,6 +109,11 @@ public Stream<T> find(Filter filter, SearchOrder order, Integer skip, Integer li
 		return find(filter, order, skip, limit, maxTime, null);
 	}
 
+	@Override
+	public Stream<T> findCloseableStream(Filter filter, SearchOrder order, Integer skip, Integer limit, int maxTime) {

```

Comment:
Not sure of the naming. findWithStreaming or something stating the main difference with the classical stream could be clearer